<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResetDimensionsDialog</class>
 <widget class="QDialog" name="ResetDimensionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Not enough GPU memory</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="iconLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="warningLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="widthLabel">
       <property name="text">
        <string>Width</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="widthSpinBox">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="nullLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="heightLabel">
       <property name="text">
        <string>Height</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="heightSpinBox">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ResetDimensionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ResetDimensionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
